src/lib/fragments/Accordion.react.js

```diff
@@ -50,39 +212,58 @@ class Accordion extends Component {
 
   renderAccordionSection(index, accordionSectionContent, sectionIsOpen) {
     const accordionHeading = this.props.accordionHeadings[index]
-      return (
-          <div className={sectionIsOpen ? "govuk-accordion__section govuk-accordion__section--expanded" : "govuk-accordion__section"}>
-            <div className="govuk-accordion__section-header">
-              <h2 className="govuk-accordion__section-heading">
-                <button className="accordion-button govuk-accordion__section-button" type="button" aria-controls={`accordion-default-content-${index}`} aria-expanded={sectionIsOpen} onClick={() => this.openOrCloseAccordionSection(index)}>
-                  <span className="govuk-accordion__section-heading-text" >
-                    <span className="govuk-accordion__section-heading-text-focus"> {accordionHeading} 
-                    </span>
-                  </span> 
-                  <span className="govuk-visually-hidden govuk-accordion__section-heading-divider"></span>
-                  <span className="govuk-accordion__section-toggle" data-nosnippet>
-                    <span className="govuk-accordion__section-toggle-focus">
-                      <span className="moj-side-navigation__item--collapsed">
-                      </span>
-                    </span>
-                    <span className= {sectionIsOpen ?  "govuk-accordion-nav__chevron" : "govuk-accordion-nav__chevron govuk-accordion-nav__chevron--down"} ></span>
-                    <span className="govuk-accordion__section-toggle-focus govuk-accordion__section-toggle-text"> {sectionIsOpen ? "Hide" : "Show"} 
-                      <span className="govuk-visually-hidden"> this section</span>
-                    </span>
+    const contentId = `accordion-default-content-${index}`;
+    return (
+      <div
+        className={sectionIsOpen ? "govuk-accordion__section govuk-accordion__section--expanded" : "govuk-accordion__section"}
+        data-section-index={index}
```

Comment:
what's this for? doesn't seem to be referenced anywhere else?

Reply:
What specifically are you referring to, the section is open as whether the sections are open and the data section index allows us to know what section we are referring to?

Reply:
I think you are right, the data section index is so that we can reference multiple elements that are  a set of selections however it's just on the high-level accordion div, I don't think it's been used at all